app/controllers/admin/export_domain_blocks_controller.rb

```diff
@@ -30,12 +30,11 @@ def import
           next if DomainBlock.rule_for(domain).present?
```

Comment:
Changes in this file seem to be reverting other work. This file should stay unchanged.